File: python/basico/examples/if_elif_else__if.py
"""
Objetivo: demostrar condicional con múltiples ramas
Referencia: if / elif / else
Tipo: keyword
Nivel: basico
"""

# imports
# no aplica

# carga de datos
nota = 7

# transformación
if nota >= 9:
    mensaje = "Sobresaliente"
elif nota >= 7:
    mensaje = "Aprobado"
else:
    mensaje = "Reprobado"

# resultado
print(mensaje)

"""output
Aprobado
"""

File: javascript/basico/examples/if_elif_else__if.js
/**
 * Objetivo: múltiples condiciones secuenciales
 * Referencia: if...else if...else
 * Tipo: keyword
 * Nivel: básico
 */

// Múltiples condiciones
const calificacion = 85;

if (calificacion >= 90) {
    console.log("Sobresaliente");
} else if (calificacion >= 80) {
    console.log("Notable");
} else if (calificacion >= 70) {
    console.log("Bien");
} else if (calificacion >= 60) {
    console.log("Aprobado");
} else {
    console.log("Suspenso");
}

// Otra aplicación
const dia = "miércoles";

if (dia === "lunes") {
    console.log("Inicio de semana");
} else if (dia === "viernes") {
    console.log("Casi fin de semana");
} else if (dia === "sábado" || dia === "domingo") {
    console.log("Fin de semana");
} else {
    console.log("Día de semana");
}

/*
output
Notable
Día de semana
*/
